mne/epochs.py

```diff
@@ -144,7 +144,7 @@ class _BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin,
     This class provides basic functionality and should never be instantiated
     directly. See Epochs below for an explanation of the parameters.
     """
-    def __init__(self, info, data, events, event_id, tmin, tmax,
+    def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
```

Comment:
I apologize as this might sound like a stupid question because of my inadequate knowledge, isn't it handled in `if event_id is None` part [here](https://github.com/mne-tools/mne-python/blob/master/mne/epochs.py#L163)?